src/components/capture/GenerateButton.tsx

```diff
@@ -142,9 +137,7 @@ function CaptureGenerateButton({
             disabled={disabled || isSaving || formActive}
             sx={buttonSx}
         >
-            <FormattedMessage
-                id={draftId ? 'cta.regenerateCatalog' : 'cta.generateCatalog'}
-            />
+            <FormattedMessage id="cta.generateCatalog.capture" />
```

Comment:
I don't think changing the button verbiage is helping anything - seems to be confusing folks.